### ISSUE TITLE
[Arducam mini 2M ov2640] Support recording video on Raspberry PI

### ISSUE THREAD

**Issue by developerkeep** [ArduCAM/RaspberryPi#17]
As the official document of ov2640 camera-chip here: https://www.uctronics.com/download/cam_module/OV2640DS.pdf, they support recording and streaming video by hardware.
So could you please support code to record or stream video using Raspberry Pi?

**Comment by ArduCAM** [comment]
It is not recommend to use OV2640 on Raspberry pi, please try to use OV5647 for Raspberry pi.

**Comment by riccardo61** [comment]
Thanks Sir, but it can be possible to install a filter for detecting only UV images on these cameras and for IR as well?

Best regards

Riccardo Tarelli

**Comment by ArduCAM** [comment]
Yes, you can change the lens and use other UV or IR filter on the lens.

**Comment by riccardo61** [comment]
Thx Lee for your clarification, can also magnify the capability of viewing of these microcameras? What is the kit and max field of view thereof?
Have good time,
Riccardo
--
Inviato da Libero Mail per Android venerdì, 04 maggio 2018, 05:08PM +02:00 da Lee  notifications@github.com :
